bookkeeper-server/src/main/java/org/apache/bookkeeper/net/NetUtils.java

```diff
@@ -68,18 +66,13 @@ public static String normalizeHostName(String name) {
         return hostNames;
     }
 
-    public static String resolveNetworkLocation(DNSToSwitchMapping dnsResolver, InetSocketAddress addr) {
-        List<String> names = new ArrayList<String>(1);
-
-        if (dnsResolver.useHostName()) {
```

Comment:
I didn't see where this logic is moved to. I think this change will also break `useHostName()`. and the tests doesn't catch this, so it means that we are missing some tests to cover this. we need to add a test to cover `useHostName`.

Reply:
Thanks, this is following the original change. 
Since BookieSocketAddress contains the hostname and InetSocketAddress, Do we still need this logic?

Reply:
OK. Thanks. I see. it is still BC issue.